cli/package.json

```diff
@@ -10,7 +10,7 @@
   "bugs": {
     "url": "https://github.com/flow-typed/flow-typed/issues/new/choose"
   },
-  "version": "3.8.0",
+  "version": "4.0.0-1",
```

Comment:
Don't bump these by hand, I do it on release with a script.

Reply:
I think this was bumped during prev releases. current RC is `4.0.0-1`